index.html

```diff
@@ -14,46 +14,190 @@
       rel="stylesheet"
     />
 
-    <title>portfolio-setup Project</title>
+    <title>portfolio-setup</title>
   </head>
   <body>
-    <!-- my-logo section -->
-    <header class="toolbar-section">
-      <div class="logo">
-        <a href="">My Logo</a>
-      </div>
-
-      <div>
-        <p><span class="iconify" data-icon="eva:menu-outline"></span></p>
-      </div>
-    </header>
-
-    <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
+    <!-------------------------------------- middle-section ---------------------------------------->
+
+    <section class="middle-section">
+      <!-- tonic section 1 -->
+      <section class="tonic-section">
+        <div class="tonic-image"></div>
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <p>CANOPY</p>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javascript</button></li>
+        </ul>
+
+        <div class="see-project">
+          <button class="btn">See Project</button>
+        </div>
```

Comment:
- [ ] I think it would be nice to add button interactions for your project as specified in the requirements for this project.
You can add hover, focus, and active selectors to the buttons for cool interactions. You can read this [article](https://www.w3schools.com/cssref/sel_hover.asp) to learn how to do it.
For your project, you can find the colors and other declarations to use for buttons at this part on Figma's website:

![image](https://user-images.githubusercontent.com/77106962/143561626-c5c359ea-a508-45da-9d58-db3d9400656a.png)